scripts/yum-wrapper

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+echo "Installing additional packages is not supported. Continue [y/n]"
```

Comment:
I think we need to add more information around this e.g. Installing additional packages is not supported because the way the VM is created. The recommended method is to run applications as containers.